cli/src/commands/git/push.rs

```diff
@@ -966,25 +971,29 @@ fn find_bookmarks_targeted_by_revisions<'a>(
 ) -> Result<Vec<(&'a RefName, LocalAndRemoteRef<'a>)>, CommandError> {
     let mut revision_commit_ids = HashSet::new();
     if use_default_revset {
-        // remote_bookmarks(remote=<remote>)..@
-        let workspace_name = workspace_command.workspace_name();
-        let expression = RevsetExpression::remote_bookmarks(
-            StringPattern::everything(),
-            StringPattern::exact(remote),
-            None,
-        )
-        .range(&RevsetExpression::working_copy(workspace_name.to_owned()))
-        .intersection(&RevsetExpression::bookmarks(StringPattern::everything()));
```

Comment:
`.intersection(&RevsetExpression::bookmarks(StringPattern::everything()));` should be retained. That's probably why the test outputs differ.

Reply:
> that should definitely be in the config, since we really shouldn't do that to user revsets

No. It's the requirement of `jj git push`, which pushes bookmarks, not arbitrary commits.

Reply:
Ok, then I'll expand the documentation to mention it.